console/src/components/Common/utils/jsUtils.js

```diff
@@ -225,7 +225,10 @@ export const downloadObjectAsJsonFile = (fileName, object) => {
     : fileName + jsonSuffix;
 
   const dataString =
-    'data:' + contentType + ',' + encodeURIComponent(JSON.stringify(object));
+    'data:' +
+    contentType +
+    ',' +
+    encodeURIComponent(JSON.stringify(object, null, 4));
```

Comment:
Could you set indent size to 2 in place of 4. 4 seems a little extra spaced out

Reply:
Sure @rikinsk, I'll make that change